lib/Support/ConvertUTF.c

```diff
@@ -48,6 +48,7 @@
 
 
 #include "llvm/Support/ConvertUTF.h"
+#include "llvm/Support/Compiler.h" // HLSL Change - for LLVM_FALLTHROUGH
```

Comment:
The warning disable for GCC added below is probably not needed now since the implicit passthrough warnings shouldn't be hit anymore